CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # bedrock-ledger ChangeLog
 
+## 11.0.1 - 2020-TBD
+
+### Changed
+- Remove `bedrock-identity` and use `bedrock-account` in tests.
```

Comment:
Document breaking changes like this:
```suggestion
- **BREAKING**: Remove `bedrock-identity` and use `bedrock-account` in tests.
```

Reply:
I don't think this is a public API breaking change.